AsyncDisplayKit/ASViewController.mm

```diff
@@ -82,6 +83,13 @@ - (void)loadView
   _node.frame = frame;
   _node.autoresizingMask = autoresizingMask;
   self.view = view;
+  
+  // ensure that self.node has a valid trait collection before a subclass's implementation of viewDidLoad.
+  // Any subnodes added in viewDidLoad will then inherit the proper environment.
```

Comment:
From what I could tell, `traitCollectionDidChange:` is not called by UIKit until after `viewDidLoad`. However `self.traitCollection` is valid in `viewDidLoad` in UIKit. To try to be consistent, I wanted to make sure that anyone that subclasses `ASViewController` will have a `self.node` with a valid `asyncTraitCollection` in `viewDidLoad`. Without this propagation we'd have to wait for UIKit's `traitCollectionDidChange` call before self.node has a valid `asyncTraitCollection`.